### PR TITLE
React Calculator v-1: Milestone 2

### PR DESCRIPTION
**Pull request by mke2111**
### In this milestone, I implemented the following features to the calculator:

- Add a components folder with 4 files to be used each with a given react component.
- 3 files within the components directory are rendered to the App component, which in turn is rendered in the index file.
- ES6 features are widely applied.

### CODE REVIEW

File: src/Components/Result.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <div>
+      <h2>{props.value}</h2>
+    </div>
+  );
+}
+
+Result.defaultProps = {
+  value: PropTypes.string.isRequired,
+};
```
Comment (BrittanyBlake) [inline src/Components/Result.js]: - [ ] the default value of the result prop should be set as 0
Screenshot from pathwright 👇🏾 
Screen Shot 2021-03-23 at 12 45 46 AM

File: src/Components/DialPad.js [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
-function Display() {
+function DialPad() {
   return (
-    <div className="container">
-      <h1>Calculator</h1>
-      <h2>0</h2>
+    <div>
       <div className="board-row">
         <button type="button">AC</button>
         <button type="button">+/-</button>
```
Comment (BrittanyBlake) [inline src/Components/DialPad.js]: - [ ] The ButtonPanel (`DialPad`) component should render the calculator panel using the Button (`SingleButton`) component, not the html `<button/>` tag.
Screenshot of requirements 👇🏾 
Screen Shot 2021-03-23 at 12 52 48 AM

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @mke2111! 👋 
 You have done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 

****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by bellom — APPROVED** [review]
## Hi @mke2111  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**